app/manifest/_base.json

```diff
@@ -71,5 +71,6 @@
     "notifications"
   ],
   "short_name": "__MSG_appName__",
+  "version": "9.5.7",
```

Comment:
Note that we started tracking the version in `package.json` rather than the manifest, so this is in the wrong place.

This move in where the version is tracked is a part of v9.6.0, so it's not on `master` yet

Reply:
I have pushed a commit to fix this.